content/usage/overview.md

```diff
@@ -70,6 +70,15 @@ pipeline:
     image: golang:1.6
 ```
 
+For private repositories credentials can be configured by secrets
```

Comment:
I would rephrase it more like that:

For private repositories you have to provide secrets to the docker daemon, valid secrets are `REGISTRY_USERNAME`, `REGISTRY_PASSWORD` and `REGISTRY_EMAIL`.

You can set these secrets using the drone-cli:

```
drone secret add --image=my.registry.com:5000/my/repo:tag \
    octocat/hello-world REGISTRY_USERNAME octocat

drone secret add --image=my.registry.com:5000/my/repo:tag \
    octocat/hello-world REGISTRY_PASSWORD mypassword

drone secret add --image=my.registry.com:5000/my/repo:tag \
    octocat/hello-world REGISTRY_EMAIL octocat@github.com
```